torch_xla/experimental/tpu.py

```diff
@@ -135,8 +152,8 @@ def configure_topology(local_rank: int,
   """
   tpu_env = get_tpu_env()
 
-  accelerator_type = tpu_env['ACCELERATOR_TYPE']
-  if tpu_env['ACCELERATOR_TYPE'].startswith('v4'):
+  accelerator_type = tpu_env[xenv.ACCELERATOR_TYPE]
+  if tpu_env[xenv.ACCELERATOR_TYPE].startswith('v4'):
```

Comment:
Let's change this to `tpu.version() == 4`

Reply:
Done